docs/release.md

```diff
@@ -0,0 +1,8 @@
+# Release
+
+## Steps to cut a release
+
+1. Bump the version
+2. Commit and open a pull request.
+3. Create a tag after that pull request gets squashed onto master. `git tag -a v0.11.1`
+4. Push the tag. `git push origin refs/tags/v0.11.1`
```

Comment:
I have my personal branch set to `origin` and heptio/sonobuoy set to `upstream`. Maybe just `git push --tags`?